packages/core/src/payment/strategies/braintree/braintree-visacheckout-payment-processor.ts

```diff
@@ -1,6 +1,8 @@
 import { RequestSender } from '@bigcommerce/request-sender';
 
-import { Address, LegacyAddress } from '../../../address';
+import { LegacyAddress } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { Address } from '../../../address';
```

Comment:
You should be able to get `Address` from API package as well.

Reply:
done